playbooks/cluster_setup_ceph.yaml

```diff
@@ -60,11 +60,12 @@
                 block:
                   - name: Create partition on OSD disks with parted
                     community.general.parted:
-                      device: "{{ lvm_volumes[0].device }}"
+                      device: "{{ ceph_osd_realdisk.stdout }}"
                       number: "{{ lvm_volumes[0].device_number }}"
+                      label: gpt # Must be the same type as the partition table of the disk used
                       state: present
                       unit: MB
-                      part_end: "{{ lvm_volumes[0].device_size }}"
+                      part_start: "-{{ lvm_volumes[0].device_size }}" # Negative numbers specifie the distance from the end of the disk
```

Comment:
In this case, you are allocating your new partition with the total size of disk available?
Did you check if the filesystem needs to be refreshed?

Reply:
No the size of the new partition is defined by device_size.
I don't see why a refresh of the filesystem should be done.